modules/util/util.go

```diff
@@ -63,3 +64,52 @@ func Min(a, b int) int {
 func IsEmptyString(s string) bool {
 	return len(strings.TrimSpace(s)) == 0
 }
+
+// NormalizeEOL will convert Windows (CRLF) and Mac (CR) EOLs to UNIX (LF)
+func NormalizeEOL(input []byte) []byte {
+	var right, left, pos int
+	if right = bytes.IndexByte(input, '\r'); right == -1 {
+		return input
+	}
+	tmp := make([]byte, len(input))
+	length := len(input)
```

Comment:
Nitpickest of nitpicks :)
```
length := len(input)
tmp := make([]byte, length)
```

Reply:
It's not really different, since `len()` is not really a function but a language construct AFAIK. These are the numbers for 150,000 iterations (method9 is len before make):
```
==== Testing DOS conversion
     Method8: 733.420742ms
     Method9: 745.064906ms
==== Testing MAC conversion
     Method8: 740.482412ms
     Method9: 734.314871ms
==== Testing LINUX conversion
     Method8: 33.425224ms
     Method9: 33.219408ms
PASS
ok      speed   3.198s
```

I've attached my tests four your reference.

Reply:
@gary-kim Well, I stand corrected. I ran more tests and it seems it's slightly faster, go figure. Updated.